sgrep_lint/tests/run-lint-tests.sh

```diff
@@ -101,7 +101,7 @@ cd "${THIS_DIR}"
 PYTHONPATH=.. pytest .
 
 local_tests() {
-    SGREP="./sgrep.py"
+    SGREP="./semgrep/__main__.py"
```

Comment:
is `__main__.py` standard practice? I was expecting to see `semgrep/semgrep.py` or something

Reply:
Yeahp. If you want to run from source. __main__.py runs when you run the module. When we package this for people `semgrep` will work. See @mschwager comment above.